fastify.js

```diff
@@ -118,6 +119,7 @@ function build (options) {
     [kBodyLimit]: bodyLimit,
     [kRoutePrefix]: '',
     [kLogLevel]: '',
+    [kLogSerializers]: {},
```

Comment:
I think this should be null instead of an empty obj

```suggestion
    [kLogSerializers]: null,
```